src/less/mixins/img-responsive.less

```diff
@@ -1,9 +1,9 @@
-/* Responsive image
+/* Fluid image
```

Comment:
Rename file to `img-fluid.less` to reflect the mixin name change.

Might not even need this mixin, maybe just move the rules inside the class